### PR TITLE
Feat(aas list) seperate aas list from backend

### PR DESCRIPTION
**Pull request by GailMelanie**
# Description

Feature Branch Merge Request

### CODE REVIEW

**Review by NilsXitaso — APPROVED** [review]

File: src/app/[locale]/list/_components/AasListTableRow.tsx [hunk 1]
```diff
@@ -35,37 +33,49 @@ const tableBodyText = {
...
         setAasOriginUrl(null);
         if (listEntry.aasId) navigate.push(`/viewer/${encodeBase64(listEntry.aasId)}`);
     };
-    const env = useEnv();
 
-    const translateListText = (property: { [key: string]: string } | undefined) => {
-        if (!property) return '';
-        return property[intl.locale] ?? Object.values(property)[0] ?? '';
-    };
+    /*    const translateListText = (property: { [key: string]: string } | undefined) => {
```
Comment (JonathanXITASO) [inline src/app/[locale]/list/_components/AasListTableRow.tsx:58]: remove this or add reason, why this is commented out

File: src/app/[locale]/list/_components/AasListTableRow.tsx [hunk 2]
```diff
@@ -102,23 +112,23 @@ export const AasListTableRow = (props: AasTableRowProps) => {
...
+            </TableCell>
+            <TableCell align="left" sx={tableBodyText}>
+                <Typography>
+                    {tooltipText(aasListEntry.aasId, 35)}
                 </Typography>
-                {tooltipText(aasListEntry.aasId, 80)}
             </TableCell>
             <TableCell align="left">
-                {aasListEntry.productGroup ? (
+                {/*  {aasListEntry.productGroup ? (
```
Comment (JonathanXITASO) [inline src/app/[locale]/list/_components/AasListTableRow.tsx:131]: remove this or add reason for commented out code

File: src/app/[locale]/list/_components/filter/SelectProductType.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    const [productClass, setProductClass] = useState<ProductClass[]>([]);
+    const [filteredAasListCount,] = useState(0);
+    const intl = useIntl();
+    /**
+     * Creates the ProductClass Filter values.
+     */
+    useEffect(() => {
+        const productClasses: ProductClass[] = [];
+        if (aasList) {
+            /*            aasList.forEach((aas) => {
```
Comment (JonathanXITASO) [inline src/app/[locale]/list/_components/filter/SelectProductType.tsx:27]: remove this or add reason for commented out code

File: src/app/[locale]/list/_components/filter/SelectProductType.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+     * Applies product filter change to the list.
+     * @param event
+     */
+    const handleFilterChange = (event: SelectChangeEvent) => {
+        setProductClassFilterValue(event.target.value);
+        if (!aasList) return;
+        if (event.target.value === '') {
+            setAasListFiltered(aasList);
+        } else {
+            /*            const filteredList = aasList.filter((aas) => {
```
Comment (JonathanXITASO) [inline src/app/[locale]/list/_components/filter/SelectProductType.tsx:59]: remove this or add reason for commented out code

**Review by JonathanXITASO — APPROVED** [review]

**Pull request merged**